java/ql/src/Violations of Best Practice/Implementation Hiding/ExposeRepresentation.ql

```diff
@@ -6,8 +6,9 @@
  * @problem.severity recommendation
  * @precision high
  * @id java/internal-representation-exposure
- * @tags reliability
- *       maintainability
+ * @tags quality
+ *       reliability
```

Comment:
I'd stick this in `maintainability`+`readability` instead.

Reply:
I disagree with this one. It is a kind of "best practice" to not expose internal representation, but the reason is that it can lead to correctness problems when the internal state is mutated in an unexpected way. So I think it should stay in `correctness`. If we make a `best-practice` category in future then we could put it in that.